### PR TITLE
Refactor Schema Files to New Structure

### PR DESCRIPTION
**Pull request by PaulHuwe**
Moved image schema and datamodel files to match jwst structure. Adjusted code paths as necessary.

### CODE REVIEW

**Review by ddavis-stsci — APPROVED** [review]
LGTM

**Comment by PaulHuwe** [comment]
Since this was at @nden 's request, I will wait for her approval before merging.

**Pull request merged**
